components/TokenManagePage/ImageBox.tsx

```diff
@@ -0,0 +1,59 @@
+import Spinner from "../ui/spinner";
+import {getFetchableUrl} from "@/lib/protocolSdk/ipfs/gateway";
+import Image from "next/image";
+import React, {Fragment} from "react";
+import {useZoraManageProvider} from "@/providers/ZoraManageProvider";
+import EditButton from "@/components/TokenManagePage/EditButton";
+import {useTokenProvider} from "@/providers/TokenProvider";
+
+interface ImageBoxProps {
+    handleImageClick: () => void;
+}
+const Container = ({
+                       children,
+                       className = "",
+                   }: {
+    children: React.ReactNode;
+    className?: string;
+}) => (
+    <div className={`size-full flex justify-center ${className}`}>{children}</div>
+);
```

Comment:
This container is redundant. Please remove it and move the `div` directly into the `ImageBox` component definition.